spark/src/test/scala/io/delta/sql/parser/DeltaSqlParserSuite.scala

```diff
@@ -260,11 +266,8 @@ class DeltaSqlParserSuite extends SparkFunSuite with SQLHelper {
     checkCloneStmt(parser, source = "t1", target = "t1", versionAsOf = Some(1L))
     // Clone with 3L table (only useful for Iceberg table now)
     checkCloneStmt(parser, source = "local.iceberg.table", target = "t1", sourceIs3LTable = true)
-    // Yet target cannot be a 3L table yet
-    intercept[ParseException] {
-      checkCloneStmt(parser, source = "local.iceberg.table", target = "catalog.delta.table",
-        sourceIs3LTable = true)
-    }
+    checkCloneStmt(parser, source = "local.iceberg.table", target = "delta.table",
```

Comment:
In https://github.com/delta-io/delta/blob/master/spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala#L287, the catalog is ignored. We need to fix it later.

Reply:
Could you please create a github issue?

Reply:
Sure https://github.com/delta-io/delta/issues/1992